bom/pom.xml

```diff
@@ -61,7 +61,7 @@
       <dependency>
         <groupId>com.datastax.oss</groupId>
         <artifactId>native-protocol</artifactId>
-        <version>1.4.10-SNAPSHOT</version>
+        <version>1.4.10-alpha1</version>
```

Comment:
This si temporary, to allow spring-boot builds relying on artifacts not released yet.